src/FormBuilder/resources/js/form-builder/src/containers/BlockEditorContainer.tsx

```diff
@@ -34,7 +41,7 @@ export default function BlockEditorContainer() {
     parseMissingBlocks(blocks);
 
     return (
-        <BlockEditorProvider value={blocks} onInput={dispatchFormBlocks} onChange={dispatchFormBlocks}>
```

Comment:
Was `onInput` causing an issue?

Reply:
I think `onInput` is responsible for the interacting with block setting inputs, so this may actually be needed to update the form